.github/workflows/markdownlint.yml

```diff
@@ -13,5 +13,4 @@ jobs:
         run: sudo gem install mdl
 
       - name: Lint markdown
-        continue-on-error: true
-        run: mdl $(find . -name *.md)
+        run: .github/workflows/lazy-lint.bash
```

Comment:
Hmm... this no longer invokes `mdl`, only prints the list of files not in the shame list, or am I not understanding it? :grimacing:

Reply:
Not at all! Thanks for noticing. Clearly that is why I didn't mark it as ready for review earlier. :laughing: